certified-connectors/Exasol/apiDefinition.swagger.json

```diff
@@ -0,0 +1,524 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This connector enables you to work with the Exasol Database.",
+    "title": "Exasol",
+    "contact": {
+      "name": "Exasol",
+      "url": "https://github.com/exasol/exasol-rest-api/issues"
+    },
+    "license": {
+      "name": "MIT License",
+      "url": "https://github.com/exasol/exasol-rest-api/blob/main/LICENSE"
+    },
+    "version": "0.1.0"
+  },
+  "host": "exasol.com",
+  "basePath": "/api/v1",
+    "x-ms-connector-metadata": [
+      {
+        "propertyName": "Website",
+        "propertyValue": "https://www.exasol.com/"
+      },
+      {
+        "propertyName": "Privacy policy",
+        "propertyValue": "https://www.exasol.com/privacy-policy"
+      },
+      {
+        "propertyName": "Categories",
+        "propertyValue": "Data;Business Intelligence"
+      }
+    ],
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/query/{query}": {
+      "get": {
+        "security": [
+          {
+            "ApiKeyAuth": []
```

Comment:
@pj-spoelders Adding a security key to each or one operation is not required and allowed. Can you please remove it.

Reply:
I have removed these.